clients/cloud/c/README.md

```diff
@@ -0,0 +1,122 @@
+# Overview
+
+Produce messages to and consume messages from [Confluent Cloud](https://www.confluent.io/confluent-cloud/) using the C client [librdkafka](https://github.com/edenhill/librdkafka).
+
+
+# Prerequisites
+
+* [librdkafka](https://github.com/edenhill/librdkafka) installed on your machine, see instructions below.
+* Access to a [Confluent Cloud](https://www.confluent.io/confluent-cloud/) cluster
+* Local file with configuration parameters to connect to your Confluent Cloud instance ([how do I find those?](https://docs.confluent.io/current/cloud/using/config-client.html#librdkafka-based-c-clients)). Format the file as follows:
+
+
+# librdkafka installation instructions
```

Comment:
@edenhill since you challenged me to challenge you...

My only concern with putting librdkafka install instructions here is that if it ever changes, we have to update in multiple places (i.e., https://github.com/edenhill/librdkafka as well as here).  My general preference is to de-dupe by putting clear librdkafka install instructions at the mothership GH repo and just provide a pointer to that from here.

Thoughts?

Reply:
Must keep mum shut.

What do you mean by mothership GH, librdkafka README?

I sort of agree, but also don't, the instructions here is a simplification of all the options available, catering to 97% of users. Linking to librdkafka installation instructions might be confusing, and they are not complete..

Reply:
Ideally, the text below would be posted verbatim at https://github.com/edenhill/librdkafka/blob/master/README.md with a linkable bookmark (i.e., https://github.com/edenhill/librdkafka/blob/master/README.md#installation)

----------

```
On Mac OSX, install librdkafka with homebrew:

  ```bash
 $ brew install librdkafka
 ```

  On Debian and Ubuntu, install librdkafka from the Confluent APT repositories,
 see instructions [here](https://docs.confluent.io/current/installation/installing_cp/deb-ubuntu.html#get-the-software) and then install librdkafka:

  ```bash
 $ apt install librdkafka-dev
 ```

  On RedHat, CentOS, Fedora, install librdkafka from the Confluent YUM repositories, instructions [here](https://docs.confluent.io/current/installation/installing_cp/rhel-centos.html#get-the-software) and then install librdkafka:

  ```bash
 $ yum install librdkafka-devel
 ```

Reply:
Okay, fair point, I shall make it happen.